examples/README.md

```diff
@@ -117,16 +101,44 @@ We have prepared a set of tutorials for an easy introduction:
   - Custom Distance implementation 
   - Custom Distance in clustering
 
-### 12. [AutoML](https://github.com/tinkoff-ai/etna/tree/master/examples/12-automl.ipynb)
-- Hyperparameters tuning
-  - How `Tune` works
-  - Example
-- General AutoML
-  - How `Auto` works
-  - Example
+#### [Feature selection](https://github.com/tinkoff-ai/etna/tree/master/examples/207-feature_selection.ipynb)
```

Comment:
Will we fix the links to point to the new repository?

Reply:
I have already fixed links in master: #47. We will do that in `docs-prototype` branch after merging current tasks I think.